### DISCUSSION TITLE
НОВАЯ ИДЕЯ

### DISCUSSION THREAD

**Discussion by godscin** [octothorp-team/public-dobrograd#659] [Предложения]
Добавьте рюкзаки,например 1 рюкзак маленький,цвет синий (10л), 2 рюкзак,вместительный,цвет фиолетовый (25л) и 3 рюкзак походный,красный цвет (50л)

Цены: 1 рюкзак-8500
           2 рюкзак-15.000
           3 рюкзак-20.000

**Comment by KOwOMei** [comment]
Уже давно писали, что ими займутся после добавления системы аксессуаров.

**Comment by LENFYS** [comment]
За
